test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionController::TestCase
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should not create new user with incorrect data" do
+    assert_no_difference 'User.count' do
+      post :create, user: {
+                            name: "Jane",
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.](https://www.codacy.com/app/conta-srdr/forsythia/file/2129755516/issues/source?bid=3139784&fileBranchId=3147671#l12)